src/GitHub.Api/Git/Repository.cs

```diff
@@ -726,15 +726,23 @@ private void UpdateUserAndEmail()
     public struct CacheUpdateEvent
     {
         [NonSerialized] private DateTimeOffset? updatedTimeValue;
-        private string updatedTimeString;
+        public string updatedTimeString;
```

Comment:
Is this variable still needed? Seems like it was replaces with UpdatedTime?

Reply:
Unity cannot serialize `DateTime` and `DateTimeOffset` objects.
We are using `string` as a backing store because this is what Unity serializes when the domain reloads.
It is our "hard copy" so to speak.

After the domain reloads `updatedTimeValue` will be null and it will be repopulated using `updatedTimeString`.